.github/workflows/performance.yml

```diff
@@ -56,4 +56,4 @@ jobs:
                   WP_VERSION=$(grep -oP "$TESTED_UP_TO_REGEX" ./readme.txt)
                   IFS='.' read -r -a WP_VERSION_ARRAY <<< "$WP_VERSION"
                   WP_BRANCH="wp/${WP_VERSION_ARRAY[0]}.${WP_VERSION_ARRAY[1]}"
-                  ./bin/plugin/cli.js perf --ci $WP_BRANCH $PREVIOUS_RELEASE_BRANCH $CURRENT_RELEASE_BRANCH
+                  ./bin/plugin/cli.js perf --ci $WP_BRANCH $PREVIOUS_RELEASE_BRANCH $CURRENT_RELEASE_BRANCH --tests-branch $WP_BRANCH
```

Comment:
no I disagree with this one :) That's not what I suggested, I think we should always use "trunk"  for the performance tests themselves, the issue is with the base branch used in the docker container which means we'll get a recent WP install (trunk) with all the new APIs with an old Gutenberg plugin `wp/5.7` branch causing conflicts. I think @mcsf's PR solves that.